frontend/src/routes/_dashboard.tsx

```diff
@@ -285,3 +282,168 @@ function Footer() {
+    type: "project"
+  },
+  {
+    id: "srv_01GZX8DKR11A",
+    slug: "auth-service",
+    project_slug: "shop-app",
+    created_at: "2024-06-17T10:20:40Z",
+    type: "service"
+  },
+  {
+    id: "srv_01GZX9KR22B1",
+    slug: "payment-api",
+    project_slug: "pay-gateway",
+    created_at: "2024-06-17T10:35:00Z",
+    type: "service"
+  },
+  {
+    id: "srv_01GZX10TR33C",
+    slug: "comment-system",
+    project_slug: "blog-site",
+    created_at: "2024-06-17T11:00:20Z",
+    type: "service"
+  },
+  {
+    id: "srv_01GZX11FR44D",
+    slug: "file-processor",
+    project_slug: "file-uploader",
+    created_at: "2024-06-17T11:25:10Z",
+    type: "service"
+  },
+  {
+    id: "srv_01GZX12LK55E",
+    slug: "analytics-service",
+    project_slug: "shop-app",
+    created_at: "2024-06-17T11:50:45Z",
+    type: "service"
+  },
+  {
+    id: "srv_01GZX13KK66F",
+    slug: "email-notifier",
+    project_slug: "pay-gateway",
+    created_at: "2024-06-17T12:15:30Z",
+    type: "service"
+  }
+] as const;
+
+export function CommandMenu() {
+  const [open, setOpen] = React.useState(false);
+  const inputRef = React.useRef<HTMLInputElement>(null);
+  const containerRef = React.useRef<HTMLDivElement>(null);
+
+  React.useEffect(() => {
+    const handleEvent = (e: KeyboardEvent | MouseEvent) => {
+      if (
+        e instanceof KeyboardEvent &&
+        e.key === "k" &&
+        (e.metaKey || e.ctrlKey)
+      ) {
+        e.preventDefault();
+        setOpen((prev) => {
+          const newState = !prev;
+          if (newState) {
+            inputRef.current?.focus();
+          } else {
+            inputRef.current?.blur();
+          }
+          return newState;
+        });
+      }
+
+      if (
+        e instanceof MouseEvent &&
+        containerRef.current &&
+        !containerRef.current.contains(e.target as Node)
+      ) {
+        setOpen(false);
+        inputRef.current?.blur();
+      }
+    };
+
+    document.addEventListener("keydown", handleEvent);
+    document.addEventListener("mousedown", handleEvent);
+
+    return () => {
+      document.removeEventListener("keydown", handleEvent);
+      document.removeEventListener("mousedown", handleEvent);
+    };
+  }, []);
+
+  return (
+    <div ref={containerRef} className="relative w-full">
+      <div
+        onClick={() => setOpen(true)}
+        className="relative w-full flex items-center"
+      >
+        <Search size={15} className="absolute left-4 text-gray-400" />
+        <Input
+          ref={inputRef}
+          className="w-full pl-12 pr-12 my-1 text-sm rounded-md border focus-visible:outline-none focus:ring-2 focus:ring-blue-500"
+          placeholder="Search for Service, Worker, CRON, etc..."
+        />
+        <div className="absolute bg-grey/20 right-4 px-2 py-1 rounded-md flex items-center space-x-1">
+          <CommandIcon size={15} />
+          <span className="text-xs">K</span>
+        </div>
+      </div>
+
+      {open && (
```

Comment:
```suggestion
      {open && resources.length > 0 && (
```

only show the commandbar if there are results